### DISCUSSION TITLE
Disable Most Touch Features in Firmware?

### DISCUSSION THREAD

**Discussion by FOSSilizedDaemon** [pine64/OpenPineBuds#89] [Q&A]
I finally got my hands on a pair of these earbuds and am loving them, but had one quick question. I am looking through the source files to modify the touch features (mostly to disable the vast majority of them), but my C is rather rusty. Has anyone already written a patch to disable touch features by chance? Or could someone point me to where in the codebase these are implemented?

**Comment by shymega** [comment]
Maintainer note: Moved to Discussion, this is a question. Please avoid using the issue tracker for questions...

**Comment by amazonparrot** [comment]
It's been more than year, not sure if you have already found your way out, for disable the touch key you can modify the apps/main/key_handler.cpp

**Reply by FOSSilizedDaemon** [reply]
Commenting out the contents of the functions in that file worked! Thank you! I might try to submit a pull request with a more easily configurable way of doing it such as an IFDEF option in a config.h.

**Reply by shymega** [reply]
@FOSSilizedDaemon I would wait until we have CMake working. Otherwise, it'll just create a mess with those efforts. Or you could submit a PR to the `cmake` branch adding that option when we're ready.
